packages/components/src/Menu/MenuItem.tsx

```diff
@@ -167,14 +173,23 @@ const MenuItemInternal: FC<MenuItemProps> = (props) => {
     >
       <Component href={href} role="menuitem" target={target}>
         {renderedIcon}
-        <span>{children}</span>
+        <Title>
+          {children}
+          {description && (
+            <Paragraph fontSize="xsmall" color="text2">
+              {description}
+            </Paragraph>
+          )}
+        </Title>
```

Comment:
If you change the `height` property to `min-height` on [ln 79 in `MenuItemLayout.tsx`](https://github.com/looker-open-source/components/blob/lara/MenuItem-description/comp1093/packages/components/src/Menu/MenuItemLayout.tsx#L79), then you can revert back to `span` here as well as the 2 other suggestions below...
```suggestion
        <span>
          {children}
          {description && (
            <Paragraph fontSize="xsmall" color="text2">
              {description}
            </Paragraph>
          )}
        </span>
```